### ISSUE TITLE
[Request] Update php-mongodb

### ISSUE THREAD

**Issue by cognitus** [oerdnj/deb.sury.org#2112]
**Frequently asked questions**

- [x] I have read [Frequenty Asked Questions](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions)

**Is your feature request related to a problem? Please describe.**

Extension mongodb 1.18.1 (https://pecl.php.net/package/mongodb) has been released.

 It seems to be required for the newer releases of the Driver installed via `composer require mongodb/mongodb` 

```
   Problem 1
    - mongodb/mongodb is locked to version 1.18.0 and an update of this package was not requested.
    - mongodb/mongodb 1.18.0 requires ext-mongodb ^1.18.0 -> it has the wrong version installed (1.17.0).
  Problem 2
    - mongodb/mongodb 1.18.0 requires ext-mongodb ^1.18.0 -> it has the wrong version installed (1.17.0).
    - mongodb/laravel-mongodb 4.2.0 requires mongodb/mongodb ^1.15 -> satisfiable by mongodb/mongodb[1.18.0].
    - mongodb/laravel-mongodb is locked to version 4.2.0 and an update of this package was not requested.

```

**Describe the solution you'd like**
Please update the package.

**Comment by keeshoekzema** [comment]
Mongodb has released version 1.19.2 by now. Is there a way to help you update the package with a pull request somewhere or something similar?

**Comment by oerdnj** [comment]
Updated to 1.19.3
